File: foodsales_eda.R
# Load required libraries
library(readxl)
library(dplyr)
library(ggplot2)
library(tidyr)
library(lubridate)
library(corrplot)

# Load the Excel file and sheet
data <- read_excel("Sampledatafoodslaes.xlsx", sheet = "FoodSales")

# View structure
str(data)

# Shape of the dataset
cat("Rows:", nrow(data), " Columns:", ncol(data), "\n")

File: foodsales_eda.py
# Import required libraries
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns
import numpy as np

# Optional: set plot style
sns.set(style="whitegrid")

# Load the Excel file
file_path = "Sampledatafoodslaes.xlsx"
data = pd.read_excel(file_path, sheet_name="FoodSales")